### ISSUE TITLE
unescaped inline  /*[[${...}]]*/ for scripts/css

### ISSUE THREAD

**Issue by haberlu** [thymeleaf/thymeleaf#12]
At some cases when dealing with javascript, an unescaped string has to be used, what cannot achieved whith the standard
/_[[ ... ]]_/ syntax. 

some examples: 

``` html
var wordCloudData = /*[[${jsonData}]]*/;
$('body').css('background-image', /*[[${background_url}]]*/ );
```

in both cases the string should not be escaped. maybe it should be some  /_[[U${background_url}]]_/  syntax invented for getting unescaped values - the same also with the leading and closing single quotes?

there is also a forum entry about this: 
http://forum.thymeleaf.org/Spring-Javascript-and-escaped-JSON-td4024739.html

Cheers!
watz

**Comment by beamhead** [comment]
+1

**Comment by danielfernandez** [comment]
Note for future evaluation: any work in this direction would require checks for avoiding adding vulnerabilities regarding code injection.

**Comment by dagnelies** [comment]
+1

**Comment by nikevasilev** [comment]
+1

**Comment by ph4nt0ma5ter** [comment]
+1

**Comment by cv** [comment]
:+1:

**Comment by beradrian** [comment]
+1

**Comment by masa-kunikata** [comment]
+1

one way to get the object from jsonString

``` javascript
  var wordCloudData = /*[[${jsonData}]]*/ '';
  var obj = eval('(' + wordCloudData + ')');
```

**Comment by ssivanat** [comment]
+1

**Comment by danielfernandez** [comment]
This has already been implemented in `3.0.0-SNAPSHOT`

See #394, #395 and #396
